### PR TITLE
Perssistent login

### PR DESCRIPTION
**Pull request by Teahaf**

### CODE REVIEW

File: app/src/main/java/com/example/demoappforfirebase/Activity/MainActivity.kt [hunk 1]
```diff
@@ -308,8 +311,7 @@ class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelecte
 
     public override fun onStart() {
         super.onStart()
-        val currentUser = auth.currentUser
-        if (currentUser == null) {
+        if(preferencesHelper.getUserId() == ""){
```
Comment (djordjenikolic960) [inline app/src/main/java/com/example/demoappforfirebase/Activity/MainActivity.kt]: Moze da se pita isStringBlankOrEmpty() umesto == ""

Comment (Teahaf) [reply app/src/main/java/com/example/demoappforfirebase/Activity/MainActivity.kt]: :)

File: app/src/main/java/com/example/demoappforfirebase/Utils/PreferencesHelper.kt [hunk 2]
```diff
@@ -4,9 +4,20 @@ import android.content.Context
...
+        preferences.edit().putString("activeUserId", uid).apply()
+    }
+
+    fun getUserId(): String {
+        return preferences.getString("activeUserId", "").toString()
+    }
+
+    /// User Sessions
+
+
```
Comment (djordjenikolic960) [inline app/src/main/java/com/example/demoappforfirebase/Utils/PreferencesHelper.kt]: extra new line

**Review by djordjenikolic960 — CHANGES_REQUESTED** [review]
changes requested

**Review by djordjenikolic960 — APPROVED** [review]
good job

**Pull request merged**